backend/accounts.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 Shift Crypto AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package backend
+
+import (
+	"github.com/digitalbitbox/bitbox-wallet-app/backend/coins/coin"
+	coinpkg "github.com/digitalbitbox/bitbox-wallet-app/backend/coins/coin"
```

Comment:
Looks like the same package twice?

Reply:
Yeah cause there are some variables named `coin`, so `coinpkg` is used to avoid a clash. Should probably convert all package accesses to `coinpkg` soon.

Reply:
I meant there are two identical import paths: `backend/coins/coin`. Remove one.

Reply:
done

Reply:
it seems you removed it from backend/backend.go but it's still here in backend/accounts.go.